reactDashboard/public/Square avatar trimmed.svg

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd rename this file to STALogoSquare.svg or similar - tidier

Also, is STABanner2.png additional or a replacement of STABanner.png. If a replacement, bin the old one and name the new one STABanner.png; if additional, rename the new one to STABannerDark.png